languages/go/oso.go

```diff
@@ -152,6 +157,42 @@ func (o Oso) IsAllowed(actor interface{}, action interface{}, resource interface
 	}
 }
 
+/*
+Return a set of actions allowed by the given (actor, resource) combination allowed
+by the policy.
+*/
+func (o Oso) GetAllowedActions(actor interface{}, resource interface{}, allowWildcard bool) (map[interface{}]interface{}, error) {
+	results := make(map[interface{}]interface{})
```

Comment:
Go doesn't have a set type apparently (just searched for this). We could maybe return a `map[interface{}]boolean` instead? Where the keys are any values returned by the query, and the value is just true for every present value?

Reply:
Apparently `map[interface{}]struct{}` is better: https://emersion.fr/blog/2017/sets-in-go/